src/types/utils.cpp

```diff
@@ -406,3 +410,43 @@ void Utils::SetColorTableAlpha(gsl::span<COLORREF>& table, const BYTE newAlpha)
         WI_UpdateFlagsInMask(color, 0xff000000, shiftedAlpha);
     }
 }
+
+
+// Function Description:
+// - Generate a Version 5 UUID (specified in RFC4122 4.3)
+//   v5 UUIDs are stable given the same namespace and "name".
+// Arguments:
+// - namespaceGuid: The GUID of the v5 UUID namespace, which provides both
+//                  a seed and a tacit agreement that all UUIDs generated
+//                  with it will follow the same data format.
+// - name: Bytes comprising the name (in a namespace-specific format)
+// Return Value:
+// - a new stable v5 UUID
+GUID Utils::CreateV5Uuid(const GUID& namespaceGuid, const gsl::span<const std::byte>& name)
+{
+    auto correctEndianNamespaceGuid{ EndianSwap(namespaceGuid) };

```

Comment:
I assume the endian swap is because of the later cast and hash?  If so, can you add a comment?  I know of other cases where an [endian swap is required](https://blogs.msdn.microsoft.com/openspecification/2013/10/08/guids-and-endianness-endi-an-ne-ssinguid-or-idne-na-en-ssinguid/), but I just want to be certain I understand why it's being done here.